Multisig/UI/ClaimToken/ClaimSafeTokenFlow.swift

```diff
@@ -22,10 +22,15 @@ class ClaimSafeTokenFlow: UIFlow {
     }
 
     override func start() {
-        //TODO check claim availability
-        // if available show intro
-        showIntro()
-        // if not available show not available
+        //TODO remove workaround and check claim availability
+        if safe.addressValue == Address(exactly: "0xfF501B324DC6d78dC9F983f140B9211c3EdB4dc7") {
+            // if not available show not available
+            showNotAvailable()
+        } else {
+            // if available show intro
```

Comment:
i think the comment can be removed because it repeats the code verbatim, same on line 27

Reply:
it's a reminder for person adding availability check.